### PR TITLE
Fix layout of datamap page

### PR DESCRIPTION
**Pull request by allisonking**
Progress on https://github.com/ethyca/fides/issues/2976

### Code Changes

* [x] Brings in the Layout component from the datamap UI as `FixedLayout`. It seemed like the conclusion was the only way to get the fixed/inside scrolling that the datamap UI has was to use a different layout component
* [x] Brought in more of the polish UI I missed from https://github.com/ethyca/fidesplus/pull/790

### Steps to Confirm

* [ ] Hopefully everything looks right now!!!

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met - no, I'll update the ticket since I didn't dedupe the PrivacyDeclaration components here. I think we can punt on that since I need to get back to privacy notice work this sprint
* [ ] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### Description Of Changes

This branch should now have everything needed to run the datamap UI from admin UI. if anything looks different from the original datamap UI, do flag it!

### CODE REVIEW

**Review by TheAndrewJackson — APPROVED** [review]
This looks good to me! I think all of the layout issues I saw are fixed. I'm pretty sure all of the polish was transferred over. @NevilleS might catch something I missed though

File: clients/admin-ui/src/features/common/FixedLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  // NOTE: unlike the main Layout, this layout specifies a fixed height
+  // of 100vh for the page, and overflow="auto" for the main content area. This
+  // allows the content area to fitting the *viewport* height, which is a
+  // special case layout needed by the datamap spatial & table views so that we
+  // scroll within those elements instead of scrolling the overall page itself
+  //
+  // Ideally we could have one common layout used by *all* pages, but we couldn't
+  // come up with a common method here without introducing other gotchas, so having
+  // a slightly different layout was decided as the most maintainable option
+  // (see https://github.com/ethyca/fidesplus/pull/709)
```
Comment (NevilleS) [inline clients/admin-ui/src/features/common/FixedLayout.tsx:25]: You can probably remove this link now, it'll be confusing in this repo

File: clients/admin-ui/src/features/common/form/inputs.tsx [hunk 2]
```diff
@@ -79,7 +79,7 @@ const TextInput = ({
     setType(type === "password" ? "text" : "password");
 
   return (
-    <InputGroup size="sm" mr="2">
+    <InputGroup size="sm">
```
Comment (NevilleS) [inline clients/admin-ui/src/features/common/form/inputs.tsx:82]: Awesome, thanks for porting this. There might be a wider impact on other forms than when I first did this work but I'm _pretty_ confident it'll be an improvement...

**Review by NevilleS — APPROVED** [review]
LGTM, thanks @allisonking!

**Pull request merged**